internal/pkg/cli/cmd/create_test.go

```diff
@@ -81,25 +74,25 @@ func TestInteractiveCreateConfig(t *testing.T) {
 		_, err = c.Send(testhelper.Enter) // enter - config
 		assert.NoError(t, err)
 
-		_, err = c.ExpectString("Enter a name for the new config")
+		_, err = c.ExpectString("Select the target branch")
```

Comment:
Tu sa name posunulo az na koniec, kedze to dava takto vacsi zmysel vypisat najprv ostatne otazky.